en/mkdocs.yml

```diff
@@ -639,12 +639,11 @@ nav:
               - Kafka Overview: reference/connectors/kafka-connector/kafka-connector-overview.md
               - Set up Kafka: reference/connectors/kafka-connector/setting-up-kafka.md
               - Enable Security for Kafka: reference/connectors/kafka-connector/enabling-security-for-kafka.md
+              - Kafka Example: reference/connectors/kafka-connector/kafka-connector-producer-example.md
               - 3.1.x and later:
-                  - Kafka Example: reference/connectors/kafka-connector/kafka-connector-producer-example.md
```

Comment:
Isn't it more convenient to have samples for both versions? WDYT @DinithiDiaz ? 
Is the same sample doc compatible with both versions? @KalinduGandara

Reply:
Example is only valid for 3.1.x